### PR TITLE
Add Flow Type annotations

### PR DESCRIPTION
**Pull request by bassettsj**
Takes care of #7

### CODE REVIEW

File: .babelrc [hunk 1]
```diff
@@ -4,9 +4,6 @@
         "stage-0",
         "react"
     ],
-    "plugins": [
```
Comment (bassettsj) [inline .babelrc:7]: Removed this but allowed it to be there in the index.

File: .npmignore [hunk 2]
```diff
@@ -0,0 +1,7 @@
+!lib
```
Comment (bassettsj) [inline .npmignore:1]: Should remove this...

File: flow-typed/npm/moment_v2.x.x.js [hunk 3]
```diff
@@ -0,0 +1,233 @@
+// flow-typed signature: ed1bee84103bb41647d690454d292a89
```
Comment (bassettsj) [inline flow-typed/npm/moment_v2.x.x.js:1]: Installed the basic libdefs

File: package.json [hunk 4]
```diff
@@ -29,48 +29,52 @@
...
-    "babel-preset-react": "^6.3.13",
-    "babel-preset-stage-0": "^6.3.13",
-    "es5-shim": "^4.4.1",
-    "eslint": "^1.10.3",
-    "eslint-config-airbnb": "^3.1.0",
-    "eslint-plugin-react": "^3.15.0",
-    "jasmine-core": "^2.4.1",
-    "karma": "^0.13.19",
-    "karma-coverage": "^0.5.3",
+    "babel-cli": "^6.14.0",
```
Comment (bassettsj) [inline package.json:37]: Updated devDependencies

**Review by bassettsj — COMMENTED** [review]
- upgraded dependencies 
- added flow types
- added a new step to publish the .js.flow style of files.

**Comment by inv8der** [comment]
👍

**Pull request merged**